### PR TITLE
Entesb 13070 initial cut

### PR DESCRIPTION
**Pull request by apupier**
initial version with Teiid naming

it is still returning a dummy completion "demo" but allows to have CI
setup and have boilerplate code done

started from https://github.com/apupier/language-server-template-java/ and then updated naming

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rareddy) [inline .classpath:1]: Can we add all eclipse specific files to gitignore?

**Review by rareddy — COMMENTED** [review]
Not knowing much specifics looks good to me.

File: .classpath (hunk 1 above)
Comment (apupier) [reply .classpath:1]: i think it is best to add the IDE specific files, it allows a faster setup of the project. These files can be filtered in other IDEs.

I really this talk explaining why ;-)
https://www.youtube.com/watch?v=J5RzSbpNLM4&list=PLU-T8l-XOWOMOVW9RrR9Vw4kc8fGzGpd1&index=4&t=2s

Comment (rareddy) [reply .classpath:1]: ok. yeah, we typically assume only maven based builds so my comment was coming from there. But your argument is valid.

**Review by lhein — APPROVED** [review]

**Pull request merged**